src/assignmentone/Utils.java

```diff
@@ -0,0 +1,72 @@
+package assignmentone;
+
+/*
+ * This class is use to calculate tax of different items
+ */
+
+public final class Utils {
+    /*
+     * This method calculate tax of raw item
+     *
+     * @param price price  of item
+     * @return tax of raw item
+     */
+
+    private float rawTax(final float price) {
+        final float taxOfRaw = (float) ((12.5 / Constants.hundred) * price);
+        System.out.println(taxOfRaw);
+        return taxOfRaw;
+    }
+
+    /*
+     * This method calulate tax of raw item
+     *
+     * @param price price of item in real number
+     * @return tax of manufactured item
+     */
+
+    private float manufacturedTax(final float price) {
+        final float remTax = (float) ((12.5 / Constants.hundred) * price);
+        return (float) (remTax + (2.0 / Constants.hundred) * price);
+    }
+
+    /*
+     * This method calulate tax of imported item
+     *
+     * @param price price of item in real number
+     * @return tax of imported item
+     */
+
+    private float importedTax(final float price) {
+        float taxOfImport = (float) ((10.0 / Constants.hundred) * price);
```

Comment:
constant.
if similar constant not used anywhere else then decalre them as final in this class only

Reply:
ok